JS/Exercise_23/index.js

```diff
@@ -0,0 +1,9 @@
+let currentHour = Number(prompt('What is the current hour?'));
+
+if ((currentHour < 0) || (currentHour > 23)) {
+    alert('Input is incorrect!');
+} else if ((currentHour < 8) || (currentHour >= 18)) {
+    alert('Hour is outside of the office hours');
+} else if ((currentHour >= 0) || (currentHour < 18)) {
```

Comment:
You don't need another condition at last now do you?

Reply:
Lol true. Will keep that in mind.